drf_kit/views/single_nested_viewsets.py

```diff
@@ -79,6 +87,23 @@ def put(self, request, *args, **kwargs):
         return Response(data, status=status_code, headers=headers)
 
 
+class SearchSingleNestedMixin:
+    @search_action
+    def search(self, request, **kwargs):
+        obj = self.get_object()
+        if obj is None:
+            raise Http404()
+        serializer = self.get_response_serializer(obj)
+        return Response(data=serializer.data, status=status.HTTP_200_OK)
+
+
+class CachedSearchableSingleNestedMixin(SearchSingleNestedMixin, CacheResponseMixin):
+    @search_action
+    @cache_response(key_func=body_cache_key_constructor)
+    def search(self, request, *args, **kwargs):
+        return super().search(request, *args, **kwargs)
```

Comment:
It does not make much sense to have a search in a single-nested because there's just 1 item there.

Reply:
It seemed strange to me as well, but I'm considering [this view](https://github.com/nilohealth/django-nilo/blob/main/django_nilo/views/__init__.py#L83) we have at Nilo, that adds search functionality as well.

I did not want to change the behavior of other viewsets we have at Nilo, but looking now it seems to be a misuse of `SingleNested` behavior. I'm okay with removing this class from this PR, and removing that viewset from Django Nilo. Wdyt?

Reply:
Update: accepted this suggestion and removed search context from this views